src/tasks/familiarweight.ts

```diff
@@ -116,6 +116,8 @@ export const FamiliarWeightQuest: Quest = {
             $familiars`Comma Chameleon, Homemade Robot`.every((fam) => have(fam)))
         ) {
           if (!have($item`box of Familiar Jacks`)) create($item`box of Familiar Jacks`, 1);
+          useFamiliar(chooseHeaviestFamiliar());
+          const heaviestTurns = CommunityService.FamiliarWeight.actualCost();
```

Comment:
We probably don't need the extra page hits here, especially since `actualCost()` uses a `visitUrl()` iirc